prisma/schema.prisma

```diff
@@ -9,3 +6,11 @@ datasource db {
   provider = "sqlite"
   url      = env("DATABASE_URL")
 }
+
+model User {
+  id            String         @id @unique @default(uuid())
+  email         String         @unique
+  password      String
+  createdAt     DateTime       @default(now())
+  updatedAt     DateTime       @updatedAt
```

Comment:
```suggestion
  id        Int      @id @default(autoincrement())
  createdAt DateTime @db.Timestamp(6) @default(now()) @map("created_at")
  updatedAt DateTime @db.Timestamp(6) @default(now()) @updatedAt @map("updated_at")
  email        String         @unique
  password        String
```

Reply:
Why do you think it's better to have the id as an `Int` than a `String`?